Utils/CommonDriver.cs

```diff
@@ -0,0 +1,16 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OnboardTaskProjectMars.Utils
+{
+    public class CommonDriver
```

Comment:
please finish implementing this

Reply:
public class CommonDriver
    {
        public static IWebDriver driver;

        public void initialise()
        {
            // create driver
            driver = new ChromeDriver();

        }

    }

Reply:
what you have done in the hooks class